### DISCUSSION TITLE
Order. totalShippingPrice does not return the total (after discounts) price

### DISCUSSION THREAD

**Discussion by nboliver** [Shopify/storefront-api-feedback#143] [Ideas & Requests]
On Storefront API version 2022-10 the [Order.totalShippingPrice](https://shopify.dev/api/storefront/2022-10/objects/Order#field-order-totalshippingprice) is returning the price of shipping before any shipping discounts are applied. 
Since `Order.totalPrice`, **does** include discounts, it seems like `totalShippingPrice` should be the same, and return the price after discounts. 
Currently, the only way to display the shipping total without manually looping through `Order.shippingDiscountAllocations` is to display a shipping line and then a shipping discounts line: 
<img width="301" alt="image" src="https://user-images.githubusercontent.com/3094055/195160350-623b8173-ce3a-41c9-9779-3806f65bb49f.png">

_Updated:_

There also appears to be an issue with `OrderLineItem.discountedTotalPrice` on this API version. The [documentation](https://shopify.dev/api/storefront/2022-10/objects/OrderLineItem#field-orderlineitem-discountedtotalprice) description of the field says:
> The total price of the line item, including discounts, and displayed in the presentment currency.

However, it is returning the price not including discounts:
<img width="931" alt="image" src="https://user-images.githubusercontent.com/3094055/195172871-e1aba9b8-c784-4a4f-ba86-70db8c050441.png">

**Comment by corygibbons** [comment]
> There also appears to be an issue with OrderLineItem.discountedTotalPrice on this API version.

Also experiencing this.
